skrub/_expressions/_parallel_coord.py

```diff
@@ -90,14 +100,21 @@ def _prepare_obj_column(col):
     }
 
 
-def _prepare_numeric_column(col, log_scale):
+def _prepare_numeric_column(col, log_scale, is_int):
     vals = col.to_numpy()
     if log_scale:
         vals = np.log(vals)
     min_val, max_val = np.nanmin(vals), np.nanmax(vals)
     tickvals = np.linspace(min_val, max_val, 10).tolist()
```

Comment:
Maybe we could delay the conversion to list and instead keep it as a NumPy array and make the conversion after the if/else block below.